server.js

```diff
@@ -64,38 +64,38 @@
 // TASK 3 : Use mongoose instead of mongodb driver 
 
 
-const http = require('http');
```

Comment:
The code is commented out. This may indicate that the code is not currently being used or needs to be revised.